packages/caliper-core/test/worker/caliper-worker.js

```diff
@@ -161,5 +190,101 @@ describe('Caliper worker', () => {
             await worker.executeRound(mockTestMessage);
             validateCallsAndWarnings(4);
         });
+
+        [5, 10].forEach(numberOfTxs => {
+            it(`should run ${numberOfTxs} transactions and wait for completion when no errors occur`, async () => {
```

Comment:
Need tests for when an error does occur, both for duration and fixed number runs